server_utils/director_discovery.go

```diff
@@ -158,7 +178,7 @@ func LaunchPeriodicDirectorDiscovery(ctx context.Context, isDirector bool) error
 	egrp := ctx.Value(config.EgrpKey).(*errgroup.Group)
 	servers, err := doDiscovery(ctx, isDirector)
 	if err != nil {
-		log.Warningln("Failed to discover available director endpoints:", err)
+		log.Warningln("Failure to discover available director endpoints:", err)
```

Comment:
I think we should stick with "Failed" here -- it matches the phrasing in log/error statements from lines 62, 114, 129, 146 and 209.